flight/PiOS/Common/pios_mpu9250_spi.c

```diff
@@ -406,6 +406,9 @@ int32_t PIOS_MPU9250_SPI_Init(uint32_t spi_id, uint32_t slave_num, const struct
 	/* Configure the MPU9250 Sensor */
 	if (PIOS_MPU9250_Config(cfg) != 0)
 		return -2;
+		
+	/* Configure a higher SPI clock speed for faster data transfer, after configuration is done */	
+	PIOS_SPI_SetClockSpeed(dev->spi_id, 21100000);
```

Comment:
Thank you very much for the feedback.

I did only work on the MPU6000 driver, than i saw the issue with Sparky2 and thought i could be easy ported to MPU9250 and make a quick edit on github.

But the pios driver for MPU9250 is already more advanced, and at my quick look at it I haven't noticed that.

Will change this when I am back from work.